scripts/functions.sh

```diff
@@ -33,7 +33,7 @@ export INFLUX_PORT="${INFLUX_PORT:-8086}"
 export BUILD_TAG="${BUILD_TAG:-testing}"
 
 # RPC-OpenStack product release, this variable is used in the config playbooks.
-export RPC_PRODUCT_RELEASE="${RPC_PRODUCT_RELEASE:-queens}"
+export RPC_PRODUCT_RELEASE="${RPC_PRODUCT_RELEASE:-rocky}"
 
 # OSA release branch
 if [ -z ${OSA_RELEASE_BRANCH+x} ]; then
```

Comment:
It looks like the dep_update PR test is failing because it's not getting the value for ``OSA_RELEASE_BRANCH``, so it thinks it needs to be ``stable/rocky`` based on the conditionals in lines 40-45 here. Looks like this logic is flawed and needs re-work.

An alternative might be to make this PR adjust everything to use the RPC_PRODUCT_RELEASE of 'master' and have master always follow OSA master, then create a 'rocky' branch whenever OSA does a Rocky branch and adjust accordingly. That might be an easier way to maintain things.